js/assessment.js

```diff
@@ -385,6 +392,9 @@ define([
         maxScore: maxScore,
         score: score,
         scoreToPass: assessmentsConfig._scoreToPass,
```

Comment:
`scoreToPass` and `isPercentageBased` are already available in variables above.

Should `correctToPass` also be surfaced in the state?